packages/allure-jest/README.md

```diff
@@ -17,14 +17,13 @@ Currently we didn't have official solution for circus test runner. Consider usin
 ### Install dependencies
 
 ```bash
-npm i -D jest-jasmine2 allure-jasmine allure-js-commons @types/jasmine
+npm i -D jest-jasmine2@^26 allure-jasmine allure-js-commons @types/jasmine
```

Comment:
why ^26 when there is ^29 released and its working fine

Reply:
I didn't work for me